webpack.config.js

```diff
@@ -45,6 +47,21 @@ module.exports = {
     "formio-script.gitbridge.min": getScriptConfig(
       "src/helpers/FormioScript/index.gitbridge.js"
     ),
+    // Default theme
+    "formio-qld": {
+      import: path.resolve(__dirname, "src/sass/formio.form.swe.scss"),
+      filename: "./temp/[name].js",
+    },
+    // SWE theme
+    "formio-qld-swe": {
+      import: path.resolve(__dirname, "src/sass/formio.form.swe.scss"),
+      filename: "./temp/[name].js",
+    },
+    // design system theme
+    "formio-qld-ds": {
+      import: path.resolve(__dirname, "src/sass/formio.form.ds.scss"),
+      filename: "./temp/[name].js",
```

Comment:
could you explain why this file ends in .js?

Reply:
as webpack is JS centrix, even the entry point is a css file it still generates a redundant empty JS module file, so i output this file to a `temp` folder as `./temp/[name].js`
this temp folder will be deleted by the `RemovePlugin` below.
and the generated CSS file (the file we actually need) will be extracted by the `MiniCssExtractPlugin` to `dist` folder root.